tools/tck-api/src/main/resources/FeatureResults.g4

```diff
@@ -80,17 +80,17 @@ propertyMap : '{' (mapContents)? '}' ;
 
 mapContents : keyValuePair (', ' keyValuePair)* ;
 
-keyValuePair: propertyKey ':' WS? propertyValue ;
+keyValuePair: (propertyKey | ('`' propertyKey '`')) ':' WS? propertyValue ;
 
 propertyKey : SYMBOLIC_NAME ;
 
 propertyValue : value ;
 
-relationshipType : ':' relationshipTypeName ;
+relationshipType : ':' (relationshipTypeName | ('`' relationshipTypeName '`'));
 
 relationshipTypeName : SYMBOLIC_NAME ;
 
-label : ':' labelName ;
+label : ':' (labelName | ('`' labelName '`')) ;
```

Comment:
Since all these three rules resolve to the same lexer rule `SYMBOLIC_NAME`, perhaps we could simplify the change by introducing an in-between parser rule `symbolicName` that allows escaping?

Like

```
propertyKey: symbolicName ;
relationshipTypeName: symbolicName ;
labelName: symbolicName ;
symbolicName: '`' SYMBOLIC_NAME '`' ;
```